services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -883,6 +883,19 @@ OFGroupAdd getInstallRoundTripLatencyGroupInstruction(IOFSwitch sw) {
                 .setWatchGroup(OFGroup.ANY)
                 .build());
 
+        TransportPort udpPort = TransportPort.of(LATENCY_PACKET_UDP_PORT);
+        List<OFAction> latencyActions = ImmutableList.of(
+                ofFactory.actions().setField(ofFactory.oxms().udpSrc(udpPort)),
+                ofFactory.actions().setField(ofFactory.oxms().udpDst(udpPort)),
+                ofFactory.actions().output(OFPort.IN_PORT, 0xFFFFFFFF));
```

Comment:
Can `0xFFFFFFFF` be moved to a global variable?

Reply:
fixed